src/packet.rs

```diff
@@ -293,6 +293,8 @@ macro_rules! align {
 
 pub const BLOB_HEADER_SIZE: usize = align!(SIZE_RANGE.end, 8);
 
+pub const BLOB_FLAG_IS_LAST: u32 = 0x2;
```

Comment:
`BLOB_FLAG_IS_LAST_IN_SLOT` and `is_last_in_slot()` would be a bit more obvious as to what it means in my opinion. Having `blob` in the name is a bit redundant, since obviously it is the blob object we are working on.